build-info/github.com/charleskorn/kaml/build.yaml

```diff
@@ -1,6 +1,20 @@
 ---
 preBuildScript: |
+    sed -i 's/.*configureVersioning.*//g' build.gradle.kts
     sed -i 's/^dependencies {$/dependencies {\n    constraints { implementation(group = "org.ajoberstar.grgit", name = "grgit-gradle", version = "4.0.0") }/' buildSrc/build.gradle.kts
+    sed -i -e '125s/^        }$/        }\n\n        repositories {\n            maven {\n                url = uri(System.getProperty("AProxDeployUrl"))\n            }\n        }/' \
+           -e 's/^import org\.gradle\.api\.publish\.maven\.MavenPublication$/import org.gradle.api.publish.maven.MavenPublication\nimport org.gradle.api.publish.maven.plugins.MavenPublishPlugin/' \
+           -e 's/^    apply<NexusPublishPlugin>/    apply<MavenPublishPlugin>/' \
+           -e '/^    configure<NexusPublishExtension> {$/,/^    }$/d' \
+           -e '/^    afterEvaluate {$/,/^    }$/d' \
+        buildSrc/src/main/kotlin/com/charleskorn/kaml/build/PublishingConfiguration.kt
 additionalArgs:
+  - "-Pversion=0.20.0"
```

Comment:
The preBuildScript and version option are specific to this version, but the build.yaml is currently global to every version.

Reply:
Do you want to have a try to modify the build pipeline to expose the version as an environment variable? That would mean we could have -Pversion=$VERSION here and it would just automatically work. 

I don't think we want things to be hard coded per version in this repo unless there is literally no other choice.

Reply:
Actually, we have this with `ENFORCE_VERSION` already? So, I should be using that instead.